components/brave_new_tab_ui/components/default/footer/braveTalkIcon.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+export default function Icon () {
+  return (
+    <svg xmlns='http://www.w3.org/2000/svg' fill='none' viewBox='0 0 24 24'>
```

Comment:
Just FYI, you can use Leo icons directly for this kind of thing:

1. Add the icon to the `leo_icons` array in `brave/ui/webui/resources/BUILD.gn`
2. Import the Leo icon component, and use the icon:
```tsx
import Icon from '@brave/leo/react/icon'

...


<Icon name="product-brave-talk"/>
```